### PR TITLE
add cowsay

### PR DESCRIPTION
**Pull request by geekytime**
Shooting from the hip, here. No easy way to test, and this is too important to wait.

### CODE REVIEW

**Comment by bcoe** [comment]
@geekytime this is awesome, I will test.

File: server.js [hunk 1]
```diff
@@ -13,6 +14,13 @@ const parser = require('yargs')
   .command('flip <text...>', 'flip text upside down', () => {}, (argv) => {
     argv.respond(flip(argv.text.join(' ')))
   })
+  .command('cowsay <text...>', 'Say it like a cow?', () => {}, (argv)={
```
Comment (bcoe) [inline server.js:17]: `(argv)={` should be `(argv) => {`

Comment (geekytime) [reply server.js:17]: You mean _equal functions_ aren't a thing in JS anymore!? What has happened to this language?

Comment (geekytime) [reply server.js:17]: Thanks for fixing. 😸

**Pull request merged**